src/vsr.zig

```diff
@@ -550,8 +563,27 @@ pub const Header = packed struct {
         }
     }
 
-    pub fn reserved() Header {
-        var header = Header{ .command = .reserved, .cluster = 0 };
+    pub fn reserved(cluster: u32, slot: u64) Header {
+        assert(slot < config.journal_slot_count);
+
+        var header = Header{
+            .command = .reserved,
+            .cluster = cluster,
+            .op = slot,
+        };
+        header.set_checksum_body(&[0]u8{});
+        header.set_checksum();
+        assert(header.invalid() == null);
+        return header;
+    }
+
+    pub fn init_prepare(cluster: u32) Header {
+        var header = Header{
+            .cluster = cluster,
+            .size = @sizeOf(Header),
+            .command = .prepare,
+            .operation = .init,
```

Comment:
What do you think of calling the "init prepare" the "genesis prepare", like the genesis block in block chain?

I've wondered a few times about this, as init is overloaded with `init()/deinit()`. On the lsm branch, the CLI also now has `format/start` as commands, no longer `init/start`.

Some alternatives: `.alpha/.origin/.root` prepare

Reply:
Some more alternatives: `.foundation` as a hat tip. ;)

Reply:
:+1: I like `root`. Should I rename the `init` operation as well?

Reply:
Great, `root prepare` also rolls off the tongue better than `init prepare`, single syllable, plus it's more of a noun than a verb so we can refer to it in docs and communication more easily, as something special.

Yes, let's rename the `init` operation to be `root`.